### ISSUE TITLE
I recommend using Docker instead of Vagrant

### ISSUE THREAD

**Issue by AaronFriel** [LSDtopotools/LSDTT_book#2]
Hello, @fclubb and @dvalters! 

I just helped a friend with using the LSDTopoTools packages and they ran into problems with the Vagrant instructions. I noticed that the Ubuntu image they use is a bit out of date (probably not a good thing since you open up ssh with a default username and password on port 22) and use a 32-bit image.

To help remedy this, I published a Docker image [afriel/lsdtopotools](https://hub.docker.com/r/afriel/lsdtopotools) and some instructions in a GitHub repository [LSDTopoTools](https://github.com/AaronFriel/LSDTopoTools).

If you have any questions about these instructions or would like to incorporate them, I would be happy to know that more folks can benefit from them.

**Comment by simon-m-mudd** [comment]
Hi Aaron, That is great, thanks! 
However one thing that you should note on the instructions (and one of the reasons we have not made the transition to docker) is that according to the _docker for windows_ documentation, installing docker for windows kills virtualbox, rendering anything you have done in vagrant/virtualbox obsolete. 
According to @dvalters this doesn't happen using docker toolbox. However, I think docker toolbox might be limited to 32 bit if you don't mess about in your windows BIOS, which is the problem with the 32 bit vagrant version. I've not tested any of this, however.

I would be *extremely* pleased if someone set up a kitematic version so that people could just click on an "app" and start using the tools. I don't have time to do that but if you are anyone else wants to give it a go I would be extremely please!
Simon

**Comment by AaronFriel** [comment]
That is correct, Docker for Windows enables Hyper-V, which prevents Virtualbox from running. 

Vagrant, however, has a Hyper-V provider that allows it to use Hyper-V instead of Virtualbox, and this permits using Ubuntu/Precise 64-bit image as well. (I used `hashicorp/precise64`, on my machine.) I wouldn't recommend this or Docker Toolbox going forward, however.

I'm not sure what you gain by using Kitematic, I've personally never used it. The instructions for using my Docker image are quite simple:

```
docker run --rm -it -v C:\LSDTopoTools\:/LSDTopoTools afriel/lsdtopotools
```

This drops you into a shell that you can then use to work in immediately.

**Comment by simon-m-mudd** [comment]
I've sourced a spare laptop so can test this out without killing my old installations. Cheers.

**Comment by simon-m-mudd** [comment]
Spoke too soon. Docker for windows requires windows 10, which our university, and many other universities, don't use. I might try docker toolbox but I think that might lead to the 32-bit problem once again.

**Comment by simon-m-mudd** [comment]
Okay, I've got docker toolbox working (sort of) on a windows 8 machine. The syntax is a little bit different to Aaron's Docker for windows, install.sh gets a "permission denied" response, I've had to install g++ and make using apt-get install, and the drives are not syncing. However, the system is 64 bit (yay!). 

Docker toolbox has no nice pretty user interface like docker for windows so I will have to spend a lot of time reading documentation. I don't know how to fix the syncing problems, or change the memory, or do all the other stuff that needs to happen before this becomes usable. And I don't have time figure any of that out at the moment. 

If anyone (Fiona? Declan? Stuart?) wants to pick up the torch feel free.

**Comment by AaronFriel** [comment]
@simon-m-mudd Did you run this command to start the image I provided?

```
docker run --rm -it -v C:\LSDTopoTools\:/LSDTopoTools afriel/lsdtopotools
```

**Comment by simon-m-mudd** [comment]
Yes. Docker toolbox uses a mingw shell that doesn't seem to like windows paths such as `C:\LSDTopoTools\` (I created that folder...so it exists). The call with the windows path failed. 
 So I ran it with 
`docker run --rm -it -v /c/LSDTopoTools/:/LSDTopoTools afriel/lsdtopotools`
and it started up but the folders didn't sync.

**Comment by AaronFriel** [comment]
Hm. I don't have a lot of experience with Docker Toolbox. I found a reported issue that volumes outside `C:\Users` won't sync, but I'm not familiar with the tools.

**Comment by simon-m-mudd** [comment]
Yeah, it seems Docker for Windows is MUCH easier to use. I am going to keep this issue open so others can attempt to implement in different OS and using the toolbox and see if anyone can streamline the toolbox process.

**Comment by AaronFriel** [comment]
The toolbox is deprecated, I would caution against going down that path because it is unsupported by Docker going forward.
